buildsystem/cpp.cmake

```diff
@@ -88,7 +89,7 @@ function(finalize_binary target_name output_name type)
 				message(FATAL_ERROR "finalize_binary flag 'allow_no_undefined' is only valid for libraries!")
 			endif()
 
-			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize")
+			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize" AND NOT MSVC)
```

Comment:
Can you confirm or deny if `libopenage` is actually a shared library (possibly named `libopenage.so` or `libopenage.dylib` on other platforms)? If it is a shared library then the linker should complain about any missing symbols. AFAIK VS doesn't have an equivalent of `-undefined dynamic_lookup`, so no worries there. Otherwise, if it is a static library (like `libopenage.a`), I'll have to search in the MSVC docs for a flag which can do that, if one exists.